content/zh/docs/community/_index.md

```diff
@@ -247,20 +247,20 @@ MOSN 社区积极拥抱开源生态，与以下开源社区建立了良好的合
 
 MOSN 社区认证的 Committer 如下：
 
-| 姓名   | GitHub                                          | 公司      |
-| ------ | ----------------------------------------------- | --------- |
-| 田阳   | [taoyuanyuan](https://github.com/taoyuanyuan)   | 蚂蚁集团  |
-| 王发康 | [wangfakang](https://github.com/wangfakang)     | 蚂蚁集团  |
-| 白鹏   | [nejisama](https://github.com/nejisama)         | 蚂蚁集团  |
-| 曹春晖 | [cch123](https://github.com/cch123)             | 蚂蚁集团  |
-| 孙福泽 | [peacocktrain](https://github.com/peacocktrain) | Boss 直聘 |
-| 陈鹏   | [champly](https://github.com/champly)           | 多点生活  |
-| 姚昌宇 | [trainyao](https://github.com/trainyao)         | 有米科技  |
-| 邓茜   | [dengqian](https://github.com/dengqian)         | 阿里云    |
-| 黄润豪 | [glyasai](https://github.com/glyasai)           | 好雨科技  |
-| 郑泽超 | [CodingSinger](https://github.com/CodingSinger) | 字节跳动  |
-| 付建豪 | [AlphaBaby](https://github.com/alpha-baby)      | 京东      |
-| 纪卓志 | [jizhuozhi](https://github.com/jizhuozhi) | 京东物流    |
+| 姓名    | GitHub                                           | 公司      |
+|-------|--------------------------------------------------|---------|
+| 田阳    | [taoyuanyuan](https://github.com/taoyuanyuan)    | 蚂蚁集团    |
+| 王发康   | [wangfakang](https://github.com/wangfakang)      | 蚂蚁集团    |
```

Comment:
Interesting. Why the Chinese version is not aligned after the tweak?

Reply:
> Interesting. Why the Chinese version is not aligned after the tweak?

Because Chinese characters are wider than Latin letters (abort 1.5)

Reply:
I'm sorry, I can't find a best alignment method, some tools use tabs, some tools use spaces, and the effects displayed on different editors are also different, not to mention that WYSIWYG directly give up align

Reply:
![image](https://user-images.githubusercontent.com/31175030/236488306-b6cd937b-15ef-4fd9-a75e-9419d676993d.png)

Reply:
It's OK